README.md

```diff
@@ -1,9 +1,23 @@
 # Gitea Matrix Template- Turn your gitea arc-green template into the matrix
+
 ## Previews:
 
-![preview 1](matrix-preview.png "Preview")
+![preview 1](preview/homepage.png "Preview")
+
+![preview 2](preview/explore.png "Preview")
+
+![preview 3](preview/login.png "Preview")
```

Comment:
do we need ths new previews?
@TylerByte666 what do you think?

Reply:
I think just one preview, we need to show the people what it looks like

Reply:
Okbyouvalready merged ... so I now the new previes are in ... I think the best way now is you change those previws youreself in masterbranch